src/App.js

```diff
@@ -22,7 +27,7 @@ export class App extends Component {
           <MoviesList movies={movies} />
         </div>
         <div className="sidebar">
-          <NewMovie />
+          <NewMovie handleAddMovies={this.handleAddMovies} />
```

Comment:
ты добавляешь один муви в функции
```suggestion
          <NewMovie addNewMovie={this.handleAddMovies} />
```